browser/greaselion/greaselion_browsertest.cc

```diff
@@ -534,6 +534,27 @@ IN_PROC_BROWSER_TEST_F(GreaselionServiceTest,
   EXPECT_EQ(title, "Altered");
 }
 
+IN_PROC_BROWSER_TEST_F(GreaselionServiceTest, CleanShutdown) {
+  ASSERT_TRUE(InstallMockExtension());
+
+  GURL url = embedded_test_server()->GetURL("www.a.com", "/simple.html");
+  ui_test_utils::NavigateToURL(browser(), url);
+  content::WebContents* contents =
+      browser()->tab_strip_model()->GetActiveWebContents();
+  ASSERT_TRUE(content::WaitForLoadStop(contents));
```

Comment:
no need for this, `NavigateToURL` is blocking

Reply:
OK, will fix.